hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/maraero/otus-go/tree/master/hw01_hello_otus
```

```suggestion
module github.com/maraero/otus-go/hw01_hello_otus
```

`tree/master` - роутинг для UI GitHub